docs/UserGuide.md

```diff
@@ -120,6 +120,7 @@ You can add an ingredient to the Kitchen Helper for tracking, containing various
 __Format:__ `addingredient /n <INGREDIENT_NAME> /c <CATEGORY> /q <QUANTITY> /p <PRICE> /e <EXPIRY>`  
 
 * `INGREDIENT_NAME` is the name of your ingredient.
+  + `INGREDIENT NAME` can only consists of alphabet letters only.
```

Comment:
Good job on clarifying the UG. Just a suggestion, would be better to use "alphabetical letters" instead.